src/igvfd/audit/file.py

```diff
@@ -117,45 +117,30 @@ def audit_external_reference_files(value, system):
             yield AuditFailure(audit_message.get('audit_category', ''), f'{detail} {audit_message.get("audit_description", "")}', level=audit_message.get('audit_level', ''))
 
 
-@audit_checker('AlignmentFile', frame='object')
-def audit_bai_alignment_files(value, system):
+@audit_checker('IndexFile', frame='object')
+def audit_index_files_derived_from(value, system):
     '''
     [
         {
-            "audit_description": "Alignment files in bai format are expected to have their corresponding bam file in `derived_from`.",
-            "audit_category": "incorrect bam file",
+            "audit_description": "Index files in tbi format are expected to have a corresponding tsv or vcf file in `derived_from`.",
+            "audit_category": "incorrect indexed file",
```

Comment:
shouldn't this be "missing indexed file"?

Reply:
nevermind, I just realized derived_from is required for index files. But for consistency with the wording for other audits I think the category should be "unexpected indexed file" instead.

Reply:
BTW you should add this file type to the list of audit scripts in commands/make_audit_docstring.py